code/game/objects/structures/window.dm

```diff
@@ -265,7 +267,7 @@
 	switch(damage_type)
 		if(BRUTE)
 			if(damage_amount)
-				playsound(src.loc, 'sound/effects/Glasshit.ogg', 75, 1)
+				playsound(src.loc, hitsound, 75, 1)
```

Comment:
Not actually a change. hitsound is just glasshit. But I moved it to a var so the paper frames could have different sounds without having to copypasta.